src/app/static/src/app/hintVersion.ts

```diff
@@ -0,0 +1 @@
+export const currentHintVersion = "1.0.0";
```

Comment:
I originally had this in root.ts but that caused a circular dependency when using it from load actions.